src/shared/components/InputBase/InputBase.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { FormGroup, HelperText } from './InputBase.styles'
+
+export type InputBaseProps = {
+  label?: string
+  error?: boolean
+  warning?: boolean
+  helperText?: string
+  disabled?: boolean
```

Comment:
Let's also include optional `className` here. Then we should pass it to the top-most container. This way we will be later able to style those components with `styled(InputBase)` or `styled(TextField)`. This applies to all your new components